vmdb/app/models/vmdb_database.rb

```diff
@@ -49,7 +49,7 @@ def top_tables_by(sorted_by, limit = nil)
     #   MAX(timestamp).
     table_ids  = self.evm_tables.collect(&:id)
     latest_ids = VmdbMetric.where(:resource_type => "VmdbTable", :resource_id => table_ids, :capture_interval_name => "hourly").select("MAX(id) AS id").group(:resource_type, :resource_id).collect(&:id)
-    metrics    = VmdbMetric.where(:id => latest_ids).all
+    metrics    = VmdbMetric.where(:id => latest_ids).to_a
```

Comment:
Do we need `.to_a` here?  We do a `sort_by` right after?  

Is there a cheat sheet of methods in relation that will treat the relation like an array and do the "right" thing?